From 649b29a37878f8ee4bab6318a58dd4c67ed4cea3 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 15 May 2022 11:31:48 +0300
Subject: [PATCH] [SW-0] gaudi3: update rom size to 128k

rom size is increased to 128kb so change in lds file accordingly.

Change-Id: I1a36a3d75ea806bc02ecbf07e242ca2643e7fdec
---

diff --git a/arch/gaudi3/arc/arcboot_rom.lds b/arch/gaudi3/arc/arcboot_rom.lds
index 97fc414..2c3d74d 100644
--- a/arch/gaudi3/arc/arcboot_rom.lds
+++ b/arch/gaudi3/arc/arcboot_rom.lds
@@ -11,7 +11,7 @@
 	ICCM	: ORIGIN = 0x00000000, LENGTH = 256K
 	STM	: ORIGIN = 0x20000000, LENGTH = 32M
 	SPI	: ORIGIN = 0x22000000, LENGTH = 16M
-	SBROM	: ORIGIN = 0x23100000, LENGTH = 64K
+	SBROM	: ORIGIN = 0x23100000, LENGTH = 128K
 	SPSRAM	: ORIGIN = 0x23120000, LENGTH = 64K
 	FWSRAM	: ORIGIN = 0x23140000, LENGTH = 128K
 	DCCM	: ORIGIN = 0x80000000, LENGTH = 64K
